media/js/firefox/partners.js

```diff
@@ -11,4 +11,16 @@
         },
         COMMON_JS_FILES
     ]);
+
+
 })(window.Modernizr);
+
+;(function($, Mozilla) {
+
+    // initialize fx family nav
+    var fxNavConfig = {
+        primaryId: 'os', // id of primary nav item to highlight (REQUIRED)
+        subId: 'partners', // id of sub nav item to highlight (OPTIONAL)
+    };
+
```

Comment:
Just need to initialize the nav:

`Mozilla.FxFamilyNav.init(fxNavConfig);`

My ultimate documentation must not be quite as ultimate as I had hoped. :see_no_evil: